core/sms_server.py

```
@@ -0,0 +1,1422 @@
+    """
+    Get individual setting value.
+    Returns cached value if available (60s TTL).
+    """
+    try:
+        pool = await get_db_pool()
+        from core.background_workers import get_setting_value as get_setting
+        
+        value = await get_setting(pool, setting_key)
+        
+        if value is None:
+            raise HTTPException(status_code=404, detail=f"Setting '{setting_key}' not found")
```

Ruff (0.14.8)

Abstract `raise` to an inner function

(TRY301)